powerlevel9k.zsh-theme

```diff
@@ -744,9 +744,11 @@ prompt_docker_machine() {
 # GO prompt
 prompt_go_version() {
   local go_version
+  local go_path
   go_version=$(go version 2>/dev/null | sed -E "s/.*(go[0-9.]*).*/\1/")
+  go_path=$(go env GOPATH)
 
-  if [[ -n "$go_version" ]]; then
+  if [[ -n "$go_version" && "${PWD##$go_path}" != "$PWD" ]]; then
```

Comment:
Apologies, the tests were more complicated than I expected. There should now be tests for whether or not GOPATH is set, as well as whether or not PWD is a child directory of GOPATH.